CheckDrive.Web/CheckDrive.Web/Views/Cars/Index.cshtml

```diff
@@ -101,3 +122,97 @@
         <a rel='nofollow' href="cars/Details/${Id}">${Id}</a>
     </div>
 </script>
+
+<style>
+    .modal-lg {
+        max-width: 100%;
+        margin: auto;/* Ekranning 90% qismini egallaydi */
+    }
+
+    .modal-content {
+        width: 60%; /* Modal o'lchamini 80% qiling */
+        max-width: 700px; /* Maksimal o'lcham */
+    }
+
+</style>
+
+@section Scripts {
+    <script>
+        $(document).ready(function () {
+        $('#openCarCreateModal').on('click',function () {
+            $('#carmodalContent').empty();          
+            $.ajax({
+                url: '/cars/create',
+                type: 'GET',                
+                contentType: 'application/x-www-form-urlencoded; charset=UTF-8',
+                headers: {
+                    'RequestVerificationToken': $('input[name="__RequestVerificationToken"]').val()
+                },
+                success: function (data) {
+                    $('#carmodalContent').html(data);
+                    console.log("Ajax muvaffaqqiyatli bajarildi");
```

Comment:
we dont need a console logging

Reply:
that's right, it was written for testing